src/ui/sass/modules/_FilterOptions.scss

```diff
@@ -132,6 +132,8 @@
     align-items: center;
     margin-left: 22px;
     cursor: pointer;
+    word-break: break-word;
+    hyphens: auto;
```

Comment:
`hyphens: auto` is only partially supported in some modern browsers (like Chrome), confirming that's OK?

Reply:
Confirmed with Rose that it's ok!